### DISCUSSION TITLE
Setting:  Automatic copy to clipboard on mouse selection

### DISCUSSION THREAD

**Discussion by someonesdad1** [sumatrapdfreader/sumatrapdf#3308] [Ideas]
I spend all day in a mintty window with a bash shell.  mintty has an optional feature of "copy on select".  When turned on, any mouse selection made in the mintty terminal is automatically copied to the clipboard.  This is an great timesaver, especially since I might do it hundreds of times per day.  I would love to see this option added to Sumatra, as I read lots of PDFs and copy text a lot.  I use an extension for Chrome that does the same thing.

**Comment by GitHubRulesOK** [comment]
its not difficult (unless one limb challenged) to adjust the selection with mouse
 and then press CTRL+C with other hand as all windows apps do
Windows itself needs three fingers , most users are used to two and SumatraPDF Key remapping can do it in One Key, and AHK can do what is requested. However auto clicking all selections will affect the native clipboard list.
IF it was auto every time  a user clicked an area to print or zoom or open the editor it would choke the clipboard limit very quickly.
![image](https://user-images.githubusercontent.com/12754913/221735196-c0c7f796-3d30-417a-9e8f-f7b80fee14a4.png)


@kjk 
selection is to https: I would as you know like a CMDmand output so TEXT can be sent to windows CLIP along with other apps

**Comment by someonesdad1** [comment]
Here's the use case.  You left click and drag in a window.  This makes a
highlighted selection of text.  Since most of the time you want to do
something with that highlighted text and the only practical use of it in a
PDF is to copy the text to the clipboard, I'm just suggesting an option
that behaves as if you then automatically pressed ctrl-C on the mouse up
event.  It won't overwhelm anything as it just copies over whatever is in
the clipboard.  It's also a simple code change, probably only a few lines
of code.
